test/it/parser/src/main/resources/sql/unsupported/unsupported.xml

```diff
@@ -262,8 +262,6 @@
     <sql-case id="unsupported_select_case_for_opengauss_438" value="select B'101' | as result;" db-types="openGauss" />
     <sql-case id="unsupported_select_case_for_opengauss_439" value="select B'101' # as result;" db-types="openGauss" />
     <sql-case id="unsupported_select_case_for_opengauss_440" value="select B'11110' # B'0000';" db-types="openGauss" />
-    <sql-case id="unsupported_select_case_for_opengauss_443" value="select extract(quarter from timestamp '2001-02-16 20:38:40') from sys_dummy;" db-types="openGauss" />
```

Comment:
Hi @lcfliuxi, can you move these two cases to supportes.xml and add expected file.

Reply:
Ok, but is it duplicate with this?
![image](https://github.com/user-attachments/assets/f3d3d0d1-c425-408d-8d8a-e6b30ea6f459)

Reply:
> Hi @lcfliuxi, can you move these two cases to supportes.xml and add expected file.

Hi @strongduanmu, I added this, pls help review again thank you.